test/components/portal.spec.tsx

```diff
@@ -39,35 +41,36 @@ describe('<Portal />', function() {
             container
         );
 
-        await waitFor(() => expect(bodySelect(portalId)).to.have.nested.property('style.position', 'fixed'));
+        await waitFor(() => expect(portal.root).to.have.nested.property('style.position', 'fixed'));
     });
 
     it('removes the component when unmounting', async function() {
         const container = document.body.appendChild(document.createElement('div'));
-        clientRenderer.render(
+        const {result} = clientRenderer.render(
             <Portal>
                 <span data-automation-id="SPAN">Popup Body</span>
             </Portal>, container);
 
-        await waitFor(() => expect(bodySelect(portalId)).to.be.present());
+        const portal = new PortalTestDriver(result as Portal);
+
+        await waitFor(() => expect(portal.root).to.be.present());
 
         ReactDOM.unmountComponentAtNode(container);
         document.body.removeChild(container);
```

Comment:
We need to check this, seems like `afterEach(function() {clientRenderer.cleanup(); });` should clean this up. If it doesn't we need to figure out why.